app/controllers/users/registrations_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Users
   class RegistrationsController < Devise::RegistrationsController
-    before_action :confirm_two_factor_authenticated, only: [:edit, :update]
+    before_action :confirm_two_factor_authenticated, only: [:edit, :update, :destroy_confirm]
+    prepend_before_action :authenticate_scope!, only: [:edit, :update, :destroy, :destroy_confirm]
```

Comment:
this shoulnd't be needed.  I opened a bug: https://github.com/18F/identity-private/issues/530